.env.example

```
@@ -14,6 +14,13 @@ GEMINI_API_KEY=your_gemini_api_key_here
 # Alternatives: gemini-1.5-flash, gemini-1.5-pro, gemini-2.0-flash-thinking-exp
 # GEMINI_MODEL=gemini-2.0-flash
 
+# ── France Travail API (Fork FR) ───────────────────────────────────────────
+# Required for: node scripts/test-api-francetravail.mjs
+# Free API: https://francetravail.io — see docs/API-FRANCE-TRAVAIL.md
+FRANCE_TRAVAIL_CLIENT_ID=your_client_id_here
+FRANCE_TRAVAIL_CLIENT_SECRET=your_client_secret_here
+FRANCE_TRAVAIL_SCOPE=api_offresdemploiv2 o2dsoffre
```

dotenv-linter (4.0.0)

[warning] [ValueWithoutQuotes] This value needs to be surrounded in quotes

(ValueWithoutQuotes)